code/game/objects/items/kitchen.dm

```diff
@@ -55,6 +56,47 @@
 	else
 		return ..()
 
+/obj/item/kitchen/knife/poison
+	name = "venom knife"
+	icon_state = "poisonknife"
+	force = 12
+	throwforce = 15
+	throw_speed = 5
+	throw_range = 7
+	var/amount_per_transfer_from_this = 5
+	var/list/possible_transfer_amounts = list(3,5)

```

Comment:
```suggestion
	var/list/possible_transfer_amounts

```

It's actually faster to set this list's values in Initialize().

Reply:
sorry, I've been on vacation without a computer for a bit, and I still don't have access to VScode
could you make the changes needed? I can commit them if you can

Reply:
nevermind, I got it